<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTuic</class>
 <widget class="QWidget" name="EditTuic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Heartbeat</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>UDP Relay Mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="zero_rtt">
     <property name="text">
      <string>Zero RTT Handshake</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="uuid"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="heartbeat"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="password"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>UUID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="congestion_control">
     <item>
      <property name="text">
       <string>bbr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cubic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>new_reno</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Congestion Control</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="udp_relay_mode">
     <item>
      <property name="text">
       <string>native</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>quic</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="udp_over_stream">
     <property name="text">
      <string>UDP over Stream</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
